Sources/SwiftSyntax/Tokens.swift.gyb

```diff
@@ -0,0 +1,101 @@
+%{
+  from gyb_syntax_support import *
+  from gyb_syntax_support.kinds import lowercase_first_word
+  # -*- mode: Swift -*-
+  # Ignore the following admonition it applies to the resulting .swift file only
+}%
+//// Automatically Generated From Tokens.swift.gyb.
+//// Do Not Edit Directly!
+//===--- Tokens.swift -----------------------------------------------------===//
+//
+// This source file is part of the Swift.org open source project
+//
+// Copyright (c) 2014 - 2022 Apple Inc. and the Swift project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See https://swift.org/LICENSE.txt for license information
+// See https://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
+//
+//===----------------------------------------------------------------------===//
+
+fileprivate func defaultTrivia(presence: SourcePresence, trivia: Trivia) -> Trivia {
+  switch presence {
+  case .present:
+    return trivia
+  case .missing:
+    return []
+  }
+}
+
+extension TokenSyntax {
+% for token in SYNTAX_TOKENS:
+%   leading_trivia = '.space' if token.requires_leading_space else '[]'
+%   trailing_trivia = '.space' if token.requires_trailing_space else '[]'
+%   if token.is_keyword:
+  public static func `${lowercase_first_word(token.name)}`(
```

Comment:
What's wrong with `token.swift_name()`?  I feel, for example, `TokenSyntax.is(presence: .missing)` doesn't seem to be creating a new instance.